### ISSUE TITLE
Typo in the README.md

### ISSUE THREAD

**Issue by MukulKolpe** [falcosecurity/plugins#253]
**Describe the bug**

The third point of  `Registering a new Plugin` section in README.md has a minor typo in the word reccomended. It should be changed to recommended.
Can I make a quick pull request to fix this issue?

**How to reproduce it**

**Expected behaviour**

**Screenshots**

**Environment**

- Falco version:

- System info:

- Cloud provider or hardware configuration:
- OS:

- Kernel:

- Installation method:

**Additional context**

**Comment by jasondellaluce** [comment]
Yeah please, go for it. Pull requests are always welcome, no need to ask for permission!

**Comment by MukulKolpe** [comment]
Thank you, @jasondellaluce. I've raised a PR.
